.github/workflows/nestml-build.yml

```diff
@@ -136,11 +135,9 @@ jobs:
           # exclude third factor plasticity models; these will only compile successfully if code generation is as a neuron+synapse pair
           export ALL_MODEL_FILENAMES=`find models/neurons -name "*.nestml" | paste -sd " "`
           echo $ALL_MODEL_FILENAMES
-          nestml --input_path $ALL_MODEL_FILENAMES --target_path target --suffix _nestml --logging_level INFO --module_name nestml_allmodels_module
-          cd target
           echo "NEST_INSTALL = ${{ env.NEST_INSTALL }}"
-          cmake -Dwith-nest=${{ env.NEST_INSTALL }}/bin/nest-config .
-          make && make install
+          sed -i 's|%NEST_PATH%|${{ env.NEST_INSTALL }}|' tests/nest_tests/resources/nest_codegen_opts.json
```

Comment:
Is this required since the path is initialized in the `NestBuilder` class by default?

Reply:
`NestBuilder` does this also implicitly. Check [build function](https://github.com/clinssen/nestml/blob/643510f9ea6cce5081bf84ac7ae3c9ff8d0571a6/pynestml/codegeneration/nest_builder.py#L87), and the command is for compiling and building the generated code outside from the python environment.